versions/3.0.md

```diff
@@ -96,8 +96,8 @@ The available status codes are defined by [RFC7231](http://tools.ietf.org/html/r
 
 ### Format
 
-The files describing the RESTful API in accordance with this specification are represented as JSON objects and conform to the JSON standards.
-YAML, being a superset of JSON, can be used as well to represent an OAS file. 
+The files describing the RESTful API in accordance with this specification consist of JSON objects and conform to the JSON standards. An OAS file written in
+YAML, a superset of JSON, also complies with this specification. 
```

Comment:
I would say any YAML comments are not part of the definition document (just syntactically spread within it – they are ignored by a parser, right?). As far as I am aware, nothing in the specification talks about comments.